packages/dev/core/src/Materials/PBR/pbrMaterial.decalMap.ts

```diff
@@ -0,0 +1,25 @@
+import { DecalMapConfiguration } from "../material.decalMapConfiguration";
+import { PBRBaseMaterial } from "./pbrBaseMaterial";
+
+declare module "./PBRBaseMaterial" {

```

Comment:
This should be:

```suggestion
declare module "./pbrBaseMaterial" {
```

it works on your windows device because windows is not case sensitive.